client/src/dojo/game/models/CubeSlidingTutorial.ts

```diff
@@ -0,0 +1,62 @@
+import { TutorialGrid,Block } from "./TutorialGrid";
```

Comment:
I think it's not relevant to put this in dojo/game
The dojo is only used for the onchain part